Package.swift

```diff
@@ -40,7 +40,7 @@ let package = Package(
             name: "CoreStore",
             dependencies: [],
             path: "Sources",
-            exclude: ["CoreStoreBridge.h", "CoreStoreBridge.m"]
+            exclude: ["CoreStoreBridge.h", "CoreStoreBridge.m", "ObjectPublisher+Reactive.swift", "Info.plist"]
```

Comment:
Why are these exclusions needed?

Reply:
Was generating error on compiling if I didn't exclude that